packages/cloudflare/src/cli/build/index.ts

```diff
@@ -100,6 +104,34 @@ export async function build(projectOpts: ProjectOptions): Promise<void> {
   logger.info("OpenNext build complete.");
 }
 
+/**
+ * Creates a `open-next.config.ts` file for the user if it doesn't exist, but only after asking for the user's confirmation.
+ *
+ * If the user refuses an error is thrown (since the file is mandatory).
+ *
+ * @param baseDir the Next.js app root folder
+ */
+async function createOpenNextConfigIfNotExistent(baseDir: string): Promise<void> {
```

Comment:
nit: can you pass projectOpts and derive the src/dst path from that?
It will be easier when we will later move to BuildOptions (OpenNext)
Thx